GPU/Common/DrawEngineCommon.cpp

```diff
@@ -832,16 +832,10 @@ void DrawEngineCommon::SubmitPrim(const void *verts, const void *inds, GEPrimiti
 	numDrawCalls_++;
 	vertexCountInDrawCalls_ += vertexCount;
 
-	if (decOptions_.applySkinInDecode && (vertTypeID & GE_VTYPE_WEIGHT_MASK)) {
+	if ((vertTypeID & GE_VTYPE_WEIGHT_MASK) && decOptions_.applySkinInDecode) {
 		DecodeVertsStep(decoded_, decodeCounter_, decodedVerts_, &dc.uvScale);
 		decodeCounter_++;
 	}
-
-	if (prim == GE_PRIM_RECTANGLES && (gstate.getTextureAddress(0) & 0x3FFFFFFF) == (gstate.getFrameBufAddress() & 0x3FFFFFFF)) {
-		// Rendertarget == texture? Shouldn't happen. Still, try some mitigations.
```

Comment:
Wasn't this for the Ridge Racer thing where it was rendering to the target and required a flush each time to avoid weird reflections or something?  Do we handle this somewhere else now?

-[Unknown]